src/input/input.directive.ts

```diff
@@ -1,5 +1,15 @@
 import { Directive, HostBinding } from "@angular/core";
-
+/**
+ * A convinence directive for applying styling to a input element.
```

Comment:
`A convinence directive` is not necessary. 
Maybe replace that with `A component`